distribution-spec/v1.0/django-oci/report.html

```diff
@@ -917,14 +1218,16 @@ <h4 style="display: inline;" onclick="javascript:toggleOutput('output-box-10')">
       &lt;/ol&gt;
       &lt;p&gt;
         
-        The current path, &lt;code&gt;v2/vsoch/django-oci/manifests/sha256:totallywrong&lt;/code&gt;, didn&#39;t match any of these.
+          The current path, &lt;code&gt;v2/vsoch/django-oci/manifests/sha256:totallywrong&lt;/code&gt;,
+        
+        didn’t match any of these.
       &lt;/p&gt;
     
   &lt;/div&gt;
 
   &lt;div id=&#34;explanation&#34;&gt;
     &lt;p&gt;
-      You&#39;re seeing this error because you have &lt;code&gt;DEBUG = True&lt;/code&gt; in
+      You’re seeing this error because you have &lt;code&gt;DEBUG = True&lt;/code&gt; in
```

Comment:
This seems weird; encoding issues? 😬

Reply:
Huh, not sure - that's what was generated when I ran it.

Reply:
I think there is a bug with the conformance testing? With Django when you have debug mode on, it sends a full page report (as shown with that message there) and it looks like this in the report:

![image](https://user-images.githubusercontent.com/814322/144727168-b93a78f7-2bf5-48a5-9103-fea1d71a7812.png)

Should this not be a failed test? Also, this response is pretty much the same as before, it just has some slightly different characters, so I don't think this should be a blocked for this PR (which fixes a big issue of not testing on the current spec). @jdolitsky you can comment - you pinged me privately and asked me to update.

Reply:
It looks like the registry is allowed to return 404 (for not found) and this is why my test passes, and we see the 404 page above here. The other option is to respon with a bad request error (below http.StatusBadRequest) and only under that case to check the content of the response.

```go
g.Context("Error codes", func() {
			g.Specify("400 response body should contain OCI-conforming JSON message", func() {
				SkipIfDisabled(pull)
				req := client.NewRequest(reggie.PUT, "/v2/<name>/manifests/<reference>",
					reggie.WithReference("sha256:totallywrong")).
					SetHeader("Content-Type", "application/vnd.oci.image.manifest.v1+json").
					SetBody(invalidManifestContent)
				resp, err := client.Do(req)
				Expect(err).To(BeNil())
				Expect(resp.StatusCode()).To(SatisfyAny(
					Equal(http.StatusBadRequest),
					Equal(http.StatusNotFound)))
				if resp.StatusCode() == http.StatusBadRequest {
					errorResponses, err := resp.Errors()
					Expect(err).To(BeNil())

					Expect(errorResponses).ToNot(BeEmpty())
					Expect(errorCodes).To(ContainElement(errorResponses[0].Code))
				}
			})
```
So I think this response that we see is correct.

Reply:
I think any issue related to conformance should be discussed in distribution-spec repo. I agree none of this should block this PR which resolves the original issue

Reply:
Agree!